Example/YipYipSwiftExample/Classes/PagedTableVIew/CustomTableView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  DataStatusTableView.swift
+//  YipYipSwiftExample
+//
+//  Created by Rens Wijnmalen on 01/07/2020.
+//  Copyright © 2020 YipYip. All rights reserved.
+//
+
+import UIKit
+import YipYipSwift
+
+public class CustomTableView: UITableView, YipYipActivityStatusTableView {
```

Comment:
De naam CustomTableView lijkt me niet goed communiceren wat er dan anders aan is. Kan het zijn dat je per ongeluk alleen de file name hebt veranderd?

Reply:
Ik heb hier niet heel erg over na gedacht aangezien het in het example project zit.